### ISSUE TITLE
Command line client needs to take in tag and description messages

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepobouncer#143]
so the info can be filled in on the revision without further manipulation on the web side

Currently importing a file through client would require a command like this:

`3drepobouncerClient <ip> <port> <username> <pw> import <file> <db> <proj> <owner>`

I'll add an option to input parameters in the form of a json file:

`3drepobouncerClient <ip> <port> <username> <pw> import -f <json file>`

where the json file would look like this:

````javascript
{
    file: "blah", //location of the 3d model file
   database: "dbName", //name of the database
   project: "projName", //name of the project
   owner: "me", //owner of the revision (i.e. whoever uploaded it)[optional, but you'll need this for backend usage]
   dxrotate: false, //rotate the model 270 degrees around the x axis [optional, you don't really need this]
   configfile: "path to import configuration settings" //for assimp configuration changes [optional, again, you don't really need this]
   tag: "myRevision", //revision name [optional]
   desc: "this is my revision" //revision description [optional]

}
````

@thenrikie  this works for you?

**Comment by thenrikie** [comment]
yep, it looks fine to me

**Comment by carmenfan** [comment]
this is done and merged into branch v1.3

Also updated the staging area to use this version.

@thenrikie  any problems let me know!
